crates/cheatcodes/spec/src/vm.rs

```diff
@@ -1228,6 +1228,24 @@ interface Vm {
     #[cheatcode(group = Json)]
     function writeJson(string calldata json, string calldata path, string calldata valueKey) external;
 
+    // -------- Base64 --------
+
+    /// Encodes a `bytes` value to base64 string
+    #[cheatcode(group = Base64)]
+    function toBase64(bytes calldata data) external pure returns (string memory);
+
+    /// Encodes a `string` value to base64 string
+    #[cheatcode(group = Base64)]
+    function toBase64(string calldata data) external pure returns (string memory);
```

Comment:
I feel like we don't need functions for both `bytes` and `string` because `bytes` alone suffices.

Reply:
Removed

Reply:
Why is `bytes` alone preferable? My understanding is the primary use case here will be for encoding/decoding tokenURIs which are strings, so if anything it seems `string` only would be preferable to avoid unnecessary casting? However looking at existing Base64 libs (OZ and solady) they also use `bytes` input, so maybe I'm missing something

Reply:
Thought strings coerce to bytes but that's only for literals.

https://github.com/foundry-rs/foundry/pull/6830